libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart-configuration.service.ts

```diff
@@ -76,11 +76,13 @@ export class FhirChartConfigurationService {
       this.buildTimeframeAnnotation('current', months),
       this.buildTimeframeAnnotation('previous', months * 2),
```

Comment:
these should only be added if months > 0, otherwise we'll have 3 identical annotations.